### PR TITLE
psi: expose StreamTypeDescription()

### PR DESCRIPTION
**Pull request by tmm1**
Makes cli more useful by printing out descriptors of the elementary stream types instead of just their ids

### CODE REVIEW

File: psi/pmt_test.go [hunk 1]
```diff
@@ -47,6 +47,10 @@ func (es *testPmtElementaryStream) StreamType() uint8 {
 	return es.streamType
 }
 
+func (es *testPmtElementaryStream) StreamTypeDescription() string {
+	return ""
```
Comment (paulboschert) [inline psi/pmt_test.go]: Should we return some test content here like "Test Stream Description" here?

File: cli/parsefile.go [hunk 2]
```diff
@@ -226,7 +226,7 @@ func printPmt(pn uint16, pmt psi.PMT) {
 	printlnf("\tPIDs %v", pmt.Pids())
 	println("\tElementary Streams")
 	for _, es := range pmt.ElementaryStreams() {
-		printlnf("\t\tPid %v : StreamType %v", es.ElementaryPid(), es.StreamType())
+		printlnf("\t\tPid %v: StreamType %v: %v", es.ElementaryPid(), es.StreamType(), es.StreamTypeDescription())
```
Comment (paulboschert) [inline cli/parsefile.go]: Should we change from `%v` to `%s` here for `es.StreamTypeDescription()`?

Also I know it's not part of this PR but the others should be `%d`s.

File: psi/pmt_test.go (hunk 1 above)
Comment (tmm1) [reply psi/pmt_test.go]: Sure that's a good idea.

File: cli/parsefile.go (hunk 2 above)
Comment (tmm1) [reply cli/parsefile.go]: Seems easier to just leave them all as `%v` and let the runtime figure it out.

File: psi/pmt_test.go (hunk 1 above)
Comment (tmm1) [reply psi/pmt_test.go]: Done.

**Comment by tmm1** [comment]
Ping?

**Comment by guygrigsby** [comment]
It looks like Paul requested changes and those have not been made.

**Comment by tmm1** [comment]
I responded to the review comment stating that I think `%v` is simpler and preferable here. But if you feel strongly I am happy to change the format string..

**Comment by guygrigsby** [comment]
Please make the changes.

**Pull request merged**